### PR TITLE
chore_: removed GetWalletToken because mobile not use it anymore

### PR DESCRIPTION
**Pull request by vkjr**
`GetWalletToken` was used only by mobile and since [this PR](https://github.com/status-im/status-mobile/pull/21625) it is not used anymore.

So related code was removed.

### CODE REVIEW

**Review by igor-sirotin — APPROVED** [review]
Nice, looks safe 👍 
Here's the desktop side, which is also unused: https://github.com/status-im/status-desktop/blob/91048fd3ef3329a2ebd78e038047f9c2cd25a3a4/src/backend/backend.nim#L130-L132

**Comment by vkjr** [comment]
@igor-sirotin, do you know how the `codecov` works here? I'm surprised that coverage considered to be lower after code removed 🤔

**Comment by igor-sirotin** [comment]
> how the `codecov` works here?

@vkjr `patch` is passing because there's no added/changed code. And removed code you don't need to cover anything 😄 
And `project` (total) coverage shows slight decrease just because our coverage is a bit flaky because of the tests' nature. Some lines of code are sometimes executed and sometimes not. To mitigate this, we have thresholds setup:
https://github.com/status-im/status-go/blob/46f2212ea98dd735de6668940e20c451312bb88a/.codecov.yml#L19-L26

Perhaps thresholds were not enough this time.

In other words, the coverage didn't really drop. It has increased, but the flakiness wasn't on your side today 🙂

**Review by dlipicar — APPROVED** [review]

**Comment by vkjr** [comment]
@igor-sirotin, so can I merge it anyway?

**Review by qfrank — APPROVED** [review]

**Comment by igor-sirotin** [comment]
> @igor-sirotin, so can I merge it anyway?

@vkjr Sure, no blockers to merge! Anything that's not `required` is only `informational` 🙂
